py/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## Unreleased
+
+- `normalize_performance_score` stores 0 to 1 cdf score instead of weighted score for each performance score component ([#2734](https://github.com/getsentry/relay/pull/2734))
```

Comment:
Please also update the top-level changelog (this is the changelog for the python library).
```suggestion
- `normalize_performance_score` stores 0 to 1 cdf score instead of weighted score for each performance score component. ([#2734](https://github.com/getsentry/relay/pull/2734))
```

Reply:
Updated!